src/Planner/PlannerJoins.cpp

```diff
@@ -76,6 +77,23 @@ void JoinClause::dump(WriteBuffer & buffer) const
 
     if (!right_filter_condition_nodes.empty())
         buffer << " right_condition_nodes: " + dump_dag_nodes(right_filter_condition_nodes);
+
+    if (!asof_conditions.empty())
+    {
+        buffer << " asof_conditions: ";
+        size_t asof_conditions_size = asof_conditions.size();
+
+        for (size_t i = 0; i < asof_conditions_size; ++i)
+        {
+            const auto & asof_condition = asof_conditions[i];
+
+            buffer << "key_index: " << asof_condition.key_index;
+            buffer << "inequality: " << toString(asof_condition.asof_inequality);
```

Comment:
Where can we print it to quickly check if space is required?.

```suggestion
            buffer << " inequality: " << toString(asof_condition.asof_inequality);
```

Reply:
Will fix in follow up pull request.